e2e/join-us.e2e.js

```diff
@@ -2,13 +2,13 @@ import chai from 'chai';
 
 const browser = global.browser; // comes from wdio
 
-describe('Home page', function () {
+describe('Join Us page', function () {
   it('should load', function (done) {
```

Comment:
Yeah I'm more in favour of using present tense - and some of the other tips in the specs are good (although some can't really be applied to js code).